### DISCUSSION TITLE
just add other file extentions

### DISCUSSION THREAD

**Discussion by hdfsyu** [XorTroll/Goldleaf#624] [Ideas]
I want the creator to add other file extentions... like nci xcz or other stuff if u have made any patches or the author has made any updates please post it here and tell me!

**Comment by Masamune3210** [comment]
![Screenshot_20210827-182128](https://user-images.githubusercontent.com/1053504/131197592-00d0bfae-b395-41b9-8ff3-4a5816cf4dab.jpg)

**Reply by hdfsyu** [reply]
what about xci?
